content/en/blog/_posts/2023-10-19-Gateway-API-GA.md

```diff
@@ -0,0 +1,194 @@
+[v1.0.0 changelog entry][chlogv1].
+
+For a complete list of every feature for every release, or to review some of
+the releases leading up to v1.0.0, please see our [historical
+changelog][clog].
+
+### Version 1 APIs
+
+This marks the **first stable release** of Gateway API, where we are now
+publishing the following APIs as `v1` (version 1):
+
+- [GatewayClass][gwc]
+- [Gateway][gw]
+- [HTTPRoute][hr]
+
+Reaching version 1 with these APIs means we future iterations on them will be
+handled in a backwards compatible manner.
+
+### CEL Migration
+
+Gateway API [CRDs][crds] now include [CEL][cel] validation. This streamlines
+the development of validation for our resources, and reduces the overhead of
+deploying Gateway API to a cluster as it eliminates the need for our historical
+[admission webhook][admw].
+
+> **Note**: For Kubernetes `v1.25` and newer we recommend against deploying the
+> webhook, and if you had it running previously you can now uninstall it in
+> favor of the validation now provided via CEL.
+
+### Kubernetes Support
+
+Gateway API generally [supports the last 5 stable Kubernetes versions][kvs]. As
+such, this release is supported on Kubernetes clusters version `1.24` through
+`1.29` at the time of release.
+
+> **Warning**: Related to our migration to [CEL][cel] as mentioned above, we
+> don't recommend deploying on `v1.24` clusters if it can be avoided as these
+> clusters will still require the admission webhook, and that webhook is
+> considered deprecated and support for it will drop shortly after this GA
+> release when Kubernetes `v1.30` becomes available. If it's unavoidable to
+> deploy on `v1.24`, just note that you'll need to upgrade fairly soon in
+> order to be considered supported.
+
+### Getting Started
+
+If you want to try Gateway API features out for yourself but don't know where
+to get started you'll want to check out one of the 20+
+[implementations](https://gateway-api.sigs.k8s.io/implementations/) which
+we list on the Gateway API website.
+
+Gateway API itself is just an API for downstream projects to implement, so take
+some time to peruse the implementations list above and find one that seems to
+fit your needs, or that you're most familiar with.
+
+> **Note**: with this release we now have basic support for downstream
+> implementations to report the results of their Gateway API conformance test
+> runs. This means that for some implementations on the above implementations
+> page you may see a `Gateway API Conformance vX.X.X` badge present under their
+> project header. This indicates that this implementation _is conformant_ for
+> the version present in the badge, and is sending back reports of their
+> conformance to the upstream project, including details on which specific
+> features they support. Click on the badge to see a more detailed report of
+> supported features, but note that the reporting system is considered
+> experimental and the structure of these reports is subject to change.
```

Comment:
I'm going to remove this from the blog post in favor of a link to the getting started guide on Gateway API. I think we should include a statement like this on the Gateway API website though, maybe directly on the implementations page?

Reply:
sure